src/tile-board/Tileboard.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Tile from './Tile';
+
+function TileBoard() {
+  const arrayGrid = createArrayGrid();
+  
+  return (
+    <div className='container text-center mt-5'>
+      {
+        // Maps over the array grid, adding a <div> element that wraps around each row.
```

Comment:
Mapping an array in JSX is quite common, so in general I wouldn't expect a comment in situations like this.